File: VB/WindowsFormsApplication25/CustomBreadCrumbEditHandler.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraEditors

Namespace WindowsFormsApplication25
    Public Class CustomBreadCrumbEditHandler
        Inherits BreadCrumbEditHandler

        Public Sub New(ByVal owner As BreadCrumbEdit)
            MyBase.New(owner)

        End Sub
        Protected Overrides Function CreateDropDownController() As BreadCrumbSelectorDropDownController
            Return New CustomBreadCrumbSelectorDropDownController(Me)
        End Function
    End Class

End Namespace

File: CS/WindowsFormsApplication25/CustomBreadCrumbEditHandler.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraEditors;

namespace WindowsFormsApplication25 {
    public class CustomBreadCrumbEditHandler : BreadCrumbEditHandler {
        public CustomBreadCrumbEditHandler(BreadCrumbEdit owner)
            : base(owner) {

        }
        protected override BreadCrumbSelectorDropDownController CreateDropDownController() {
            return new CustomBreadCrumbSelectorDropDownController(this);
        }
    }

}
